sdk/batch/Microsoft.Azure.Batch/src/Common/Enumerations/DiffDiskPlacement.cs

```diff
@@ -0,0 +1,26 @@
+﻿// 
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+// 
```

Comment:
nit:  This looks like hanging comment.  Was this intentional?  Is L4 an extra blank?